tests/test_entities.py

```diff
@@ -107,7 +107,7 @@ def test_init_succeeds(self):
                 entities.ComputeAttribute,
                 entities.ComputeProfile,
                 entities.ConfigGroup,
-                entities.ConfigTemplate,
```

Comment:
I let this one, as here are other deprecated entities.

Reply:
hehe - I think that "see below" tries to point at:
https://github.com/SatelliteQE/nailgun/blob/master/tests/test_entities.py#L225-L228

- but some other contributors probably just replicated that comment without appending the commented entity in the deprecated list.
- Anyway, even if deprecated, the entity is kept in `entities` - taking `OperatingSystemParameter` for example.

Since you're removing the entity entirely - I would either simply delete this line here or kept the entity and appended it in the deprecated list.

I'm not 100% sure about what I'm saying here, I just assume from looking at this piece of code.
@mirekdlugosz - any comments on how to approach this one?

Reply:
@rplevka looks like deprecated entities still exist, they only raise exception when used on Satellite newer than when they existed. So if we want to call them deprecated here, we should leave them in nailgun, but with changes.

Since we are removing them from nailgun, I think we should remove them here as well.

And since we are the only consumers of nailgun, I don't see a point of going out of our way with deprecation warning etc.